server/datastore/psql/database.go

```diff
@@ -0,0 +1,30 @@
+package psql
+
+import (
+	"github.com/jackc/pgx"
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+/* Create a Struct that implements datastore.DB*/
+type DB struct {
+	Connection *pgx.Conn
```

Comment:
let's leave this variable uncapitalized, either `connection` or `conn`

The reason is because in golang, the way you show that a field is public or private (like in java) is through capitalizing the first letter. Capital first letter = public, lowercase first letter = private